exampleSite/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/CaiJimmy/hugo-theme-stack-example
```

Comment:
Is it necessary to turn `exampleSite` folder into a Go Module? 

And `github.com/CaiJimmy/hugo-theme-stack/exampleSite` might be a more appropriate name, since I don't have a repo named `hugo-theme-stack-example`.

Reply:
> Is it necessary to turn `exampleSite` folder into a Go Module?

No, it isn't.

That's the reason why I turned the `exampleSite` folder into a module: Currently you have to run:

```
hugo server --themesDir ../..
``` 

Once we turn it into a module, this command will be sufficient:

```
hugo server
```

But I can revert the change of the exampleSite to a module, if you want.

> And `github.com/CaiJimmy/hugo-theme-stack/exampleSite` might be a more appropriate name, since I don't have a repo named `hugo-theme-stack-example`

Good point, corrected now.

Reply:
Thanks for the information 👍

I'll remove `exampleSite/go.mod` as it isn't strictly necessary.